contrib/upgrade_testnet/docker-compose.yml

```diff
@@ -3,8 +3,8 @@ version: '3'
 services:
   desmosnode0:
     container_name: desmosnode0
-    image: "desmoslabs/desmos-cosmovisor:v1.0.1"
-    command: "start"
+    image: "desmoslabs/desmos-cosmovisor:v2.3.0-mainnet"
+    command: "start --x-crisis-skip-assert-invariants"
```

Comment:
This should be removed in the future. It's there so that the chain can start and the upgrade can happen properly